src/app/home/navigation.component.ts

```diff
@@ -2,22 +2,42 @@ import { Component, OnInit } from '@angular/core';
 import { UserService } from '../shared/user.service';
 import { CouchService } from '../shared/couchdb.service';
 import { Router } from '@angular/router';
+import * as languages from '../../../design/languages/languages-mockup.json';

```

Comment:
this file can go to assets or shared as it is not mockup data

Reply:
It should go into shared because it will be built into the js files Webpack outputs.  The assets folder is for files that will be included as is in the app.